internal/controller/runtime_test.go

```diff
@@ -345,3 +346,4 @@ func testSpec(t *testing.T, runtime Runtime) *corev1.PodSpec {
 		return nil
 	}
 }
+*/
```

Comment:
TODO: fix these or create some new tests, ya?

Reply:
runtime.go got replaced by resources pkg... We should recreate these in there eventually. Not attempting in this PR.